GameData/ROCapsules/PartConfigs/GeminiBDB/GeminiReentryControlSystem.cfg

```diff
@@ -0,0 +1,151 @@
+PART
+{
+
+	name = ROC-ReentryControlSystemBDB
+	module = Part
+	author = Cobaltwolf, capkirk
+
+	RSSROConfig = True
+
+	MODEL
+	{
+		model = ROCapsules/Assets/BDB/Gemini/bluedog_Gemini_ReentryRCS
+		rotation = 0, 180, 0
+	}
+
+	scale = 1
+	rescaleFactor = 1.524
+	node_stack_bottom = 0.0, -0.15536, 0.0, 0.0, -1.0, 0.0, 0
+	node_stack_top = 0.0,  0.1293, 0.0, 0.0, 1.0, 0.0, 0
+	attachRules = 1,0,1,1,0
+	TechRequired = flightControl
+	entryCost = 1500
+	cost = 360
+	category = Control
+	subcategory = 0
+	title = Gemini Re-entry Control System
+	manufacturer = McDonnell Aircraft
+	description  = This section contains fuel and thrusters to provide Gemini 3 axis attitude control after detaching the adapter section. RCS O/F Ratio 1.3:1.
+	mass = 0.133
+	dragModelType = default
+	maximum_drag = 0.20
+	minimum_drag = 0.20
+	angularDrag = 2
+	crashTolerance = 10
+	breakingForce = 50
+	breakingTorque = 50
+	skinMaxTemp = 3350
```

Comment:
Wow.  Heatshields for LEO only hit 2600.  Lunar 3600.  Meanwhile, getting something towards 3350 skin and < 900 internal is probably not going to be a thing, practically.